components/camel-smpp/src/main/java/org/apache/camel/component/smpp/SmppConsumer.java

```diff
@@ -198,6 +198,9 @@ private void reconnect() {
                 reconnectLock.unlock();
             }
         } else {
+            // TODO Update once basline is java 21
```

Comment:
typo basline --> baseline , I guess it wil lbe useful to have the correct workding to easily search for similar todos